woocommerce/woocommerce-ios
# Add localised strings to the app

This PR adds translations to the app. It contains:
 - The current revision of the strings translations.
 - The changes to the project to let it aware of the translations;
 - The scripts required to pull the translations from GlotPress.

To Test:
 - Run `Scripts/update-translations.rb`. It should pull the latest revisions of the strings (since the translations were already at 99%, it's possible that no changes are available for download).
 - Build and run the project.
 - Change the language of the device and verify that the app picks the right strings set.

Thanks a lot @jleandroperez for your help to make this work!!!!
